impeller/aiks/canvas.cc

```diff
@@ -819,7 +819,7 @@ EntityPass& Canvas::GetCurrentPass() {
 }
 
 size_t Canvas::GetClipDepth() const {
```

Comment:
I'm not sure I understand the motivation behind changing the clip_depth to clip_height. Perhaps rename the canvas call too?

Reply:
because we count up and not down maybe? but yeah I would complete the rename

Reply:
Done.

The motivation is that we have two distinct clip tracking concepts to contend with, and I think using different labels for them moving forward will curb confusion (especially while doing the work to collapse EntityPass/Aiks):
* Clip height: The placement of a clip within the clip stack. This is a detail of coverage rectangle tracking and has nothing to do with actual draw calls themselves.
* Clip depth: The depth integer assigned to all draws/clips that informs the Z position range to use for each draw call.

Reply:
SGTM. Having a "zPosition" `float` might also be clearer.